apps/api/v2/src/modules/webhooks/controllers/event-types-webhooks.controller.ts

```diff
@@ -0,0 +1,100 @@
+import { API_VERSIONS_VALUES } from "@/lib/api-versions";
+import { ApiAuthGuard } from "@/modules/auth/guards/api-auth/api-auth.guard";
+import { GetWebhook } from "@/modules/webhooks/decorators/get-webhook-decorator";
+import {
+  DeleteManyWebhooksOutputResponseDto,
+  WebhookOutputDto,
+  WebhookOutputResponseDto,
+  WebhooksOutputResponseDto,
+} from "@/modules/webhooks/outputs/webhook.output";
+import { WebhooksService } from "@/modules/webhooks/services/webhooks.service";
+import {
+  Controller,
+  Post,
+  Body,
+  UseGuards,
+  Get,
+  Param,
+  Query,
+  Delete,
+  Patch,
+  ParseIntPipe,
+} from "@nestjs/common";
+import { ApiOperation, ApiTags } from "@nestjs/swagger";
+import { Webhook } from "@prisma/client";
+import { plainToClass } from "class-transformer";
+
+import { SUCCESS_STATUS } from "@calcom/platform-constants";
+import { SkipTakePagination } from "@calcom/platform-types";
+
+import { IsUserEventTypeWebhookGuard } from "../guards/is-user-event-type-webhook-guard";
+import { CreateWebhookInputDto } from "../inputs/create-webhook.input";
+
+@Controller({
+  path: "/v2/event-types/:eventTypeId/webhooks",
```

Comment:
Since this root path is “/event-types”, it feels like this feel belongs in that folder, not webhooks.